src/commands/identity/update.ts

```diff
@@ -0,0 +1,73 @@
+import inquirer from 'inquirer'
+import { userIdPrompt } from '../../ui/prompts'
+
+import { Flags } from '@oclif/core'
+import UpdateCommand from '../updateCommand'
+import { fetchUserProfile, updateUserProfile } from '../../api/userProfile'
+
+export default class UpdateIdentity extends UpdateCommand {
+    static hidden = false
+    authRequired = true
+    static description = 'Update your DevCycle Identity.'
+
+    prompts = [
+        userIdPrompt
+    ]
+
+    static args = {
+        ...UpdateCommand.args
+    }
+
+    static flags = {
+        userId: Flags.string({
+            name: 'userId',
+            description: 'DevCycle Identity User ID to be used for Overrides & Self-Targeting'
+        }),
+        ...UpdateCommand.flags,
+    }
+
+    public async run(): Promise<void> {
+
+        const { flags } = await this.parse(UpdateIdentity)
+        const { headless, project } = flags
+        await this.requireProject(project, headless)
+
+        const { dvcUserId: existingUserId } = await fetchUserProfile(this.authToken, this.projectKey)
+
+        let dvcUserId
+        let confirmation = false
+        if (existingUserId) {
+            const { confirmUpdate } = await inquirer.prompt([{
+                name: 'confirmUpdate',
+                message: `
+                    A User ID (${existingUserId}) is associated with your account for this project
+                    (${this.projectKey}). Would you like to update it?
+                `,
+                type: 'confirm'
+            }])
+            confirmation = confirmUpdate
+        } else {
+            confirmation = true
+        }
+
+        if (!confirmation) {
+            this.writer.printCurrentValues(
+                `No DevCycle Identity changes made for project: ${this.projectKey} and User ID: ${existingUserId}`
+            )
+            return
+        }
+
+        if (!flags.userId) {
+            const input = await inquirer.prompt([userIdPrompt])
+            dvcUserId = input.userId
```

Comment:
I think we should add or indicate some kind of warning if a user enters a blank dvc id? I was able to just enter and it wiped by id (and all overrides)